indico/modules/events/editing/schemas.py

```diff
@@ -139,6 +140,29 @@ class Meta:
     review_conditions_valid = fields.Boolean()
 
 
+class EditableBasicSchema(mm.ModelSchema):
+    class Meta:
+        model = Editable
+        fields = ('id', 'type', 'state', 'editor')
+
+    state = fields.String()
+    editor = fields.Nested(UserSchema, only=('id', 'avatar_bg_color', 'full_name'))
```

Comment:
Currently, Assigning of editor is not implemented so it will give error.

Reply:
Why does it give an error? If there is no editor assigned it should serialize as `None`

Reply:
Resolved!